webpack.config.js

```diff
@@ -36,11 +36,6 @@ const config = {
             exclude: /node_modules/,
             use: [{
                 loader: 'ts-loader',
-                options: {
```

Comment:
not sure about that one. Why do you remove it?

Reply:
According to the default comment, those options override the typescript config.

Although I am not sure entirely why, overriding the typescript config causes typescript to be unable to find modules like vscode and vscode-languageclient, despite the fact that they are installed. 

![image](https://user-images.githubusercontent.com/20326645/62372145-b86a9480-b504-11e9-9dc0-d102f6a04845.png)

I know that the third error in that screenshot is masked by the current typescript config. I suspect that the current typescript config is what allows vscode and vscode-languageclient to be found? This error only happens with those packages only. 

Also, since vscode-xml didn't have those options, I just deleted them.